src/actions/index.js

```diff
@@ -0,0 +1,26 @@
+function createCreator(book) {
+  return {
+    type: 'CREATE_BOOK',
+    payload: {
+      id: book.id,
+      title: book.title,
+      category: book.category,
+    },
+  };
+}
+
+function removeCreator(book) {
+  return {
+    type: 'REMOVE_BOOK',
+    payload: {
+      id: book.id,
+      title: book.title,
+      category: book.category,
```

Comment:
You could pass only the `book.id` here, you don't need the title and the category.